guessing-game-v2/src/components/MetaMaskConnectButton.vue

```diff
@@ -1,90 +1,100 @@
 <template>
-  <el-button @click="connectMetamask" size="large" >
+  <el-button @click="connectMetamask" size="large">
     <img src="@/assets/icons/icon_metamask.png" alt="Connect with wallet" class="metamask-icon" />
     {{ buttonText }}
   </el-button>
 </template>
 
 <script>
-import detectEthereumProvider from '@metamask/detect-provider';
-import {useWalletStore} from "@/stores/walletStore";
-import { useMessageStore } from '@/stores/messageStore';
-import { ref } from 'vue';
-import Web3listener from "@/lib/web3listener";
-
+import detectEthereumProvider from '@metamask/detect-provider'
+import { useWalletStore } from '@/stores/walletStore'
+import { useMessageStore } from '@/stores/messageStore'
+import { ref } from 'vue'
+import Web3listener from '@/lib/web3listener'
+import { trackEvent } from '../lib/utils'
 
 export default {
   name: 'MetaMaskConnectButton',
   setup() {
-    const walletStore = useWalletStore();
-    const messageStore = useMessageStore();
-    const buttonText = ref("Connect with MetaMask");
+    const walletStore = useWalletStore()
+    const messageStore = useMessageStore()
+    const buttonText = ref('Connect with MetaMask')
```

Comment:
Likely helpful to have the same JS style everywhere. Perhaps leveraging the linters for this would be a good choice ?

Reply:
We have a config for this: https://github.com/ten-protocol/sample-applications/blob/6d335dfd4fbccb3bd4d62d0d38a86639d681b2be/guessing-game-v2/.prettierrc.json#L5

That's what my Prettier extension uses